axon-framework/modules.md

```diff
@@ -83,18 +83,18 @@ Besides main modules, there are several extension modules which complement Axon
 
 | Module | Artifact Id | Group Id | Maven Central |
 | :--- | :--- | :--- | :---: |
-| [Axon AMQP](maven-dependencies.md#axon-amqp) | axon-amqp | org.axonframework.extensions.amqp | [available](https://search.maven.org/search?q=a:axon-amqp) |
-| [Axon AMQP Spring Boot Starter](maven-dependencies.md#axon-amqp-spring-boot-starter) | axon-amqp-spring-boot-starter | org.axonframework.extensions.amqp | [available](https://search.maven.org/search?q=a:axon-amqp-spring-boot-starter) |
-| [Axon Kafka](maven-dependencies.md#axon-kafka) | axon-kafka | org.axonframework.extensions.kafka | [available](https://search.maven.org/search?q=a:axon-kafka) |
-| [Axon Kafka Spring Boot Starter](maven-dependencies.md#axon-kafka-spring-boot-starter) | axon-kafka-spring-boot-starter | org.axonframework.extensions.kafka | [available](https://search.maven.org/search?q=a:axon-kafka-spring-boot-starter) |
-| [Axon Spring Cloud](maven-dependencies.md#axon-spring-cloud) | axon-springcloud | org.axonframework.extensions.springcloud | [available](https://search.maven.org/search?q=a:axon-springcloud) |
-| [Axon Spring Cloud Spring Boot Starter](maven-dependencies.md#axon-jgroups-spring-boot-starter) | axon-springcloud-spring-boot-starter | org.axonframework.extensions.springcloud | [available](https://search.maven.org/search?q=a:axon-springcloud-spring-boot-starter) |
-| [Axon JGroups](maven-dependencies.md#axon-jgroups) | axon-jgroups | org.axonframework.extensions.jgroups | [available](https://search.maven.org/search?q=a:axon-jgroups) |
-| [Axon JGroups Spring Boot Starter](maven-dependencies.md#axon-jgroups-spring-boot-starter) | axon-jgroups-spring-boot-starter | org.axonframework.extensions.jgroups | [available](https://search.maven.org/search?q=a:axon-jgroups-spring-boot-starter) |
-| [Axon Mongo](maven-dependencies.md#axon-mongo) | axon-mongo | org.axonframework.extensions.mongo | [available](https://search.maven.org/search?q=a:axon-mongo) |
-| [Axon CDI](maven-dependencies.md#axon-cdi) | axon-cdi | org.axonframework.extensions.cdi | [available](https://search.maven.org/search?q=a:axon-cdi) |
-| [Axon Tracing](maven-dependencies.md#axon-tracing) | axon-tracing | org.axonframework.extensions.tracing | [available](https://search.maven.org/search?q=a:axon-tracing) |
-| [Axon Tracing Spring Boot Starter](maven-dependencies.md#axon-tracing-spring-boot-starter) | axon-tracing-spring-boot-starter | org.axonframework.extensions.tracing | [available](https://search.maven.org/search?q=a:axon-tracing-spring-boot-starter) |
+| [Axon AMQP]() | axon-amqp | org.axonframework.extensions.amqp | [available](https://search.maven.org/search?q=a:axon-amqp) |
```

Comment:
Missing the links again here. Was this intentional?

Reply:
Corrected